pkg/validate/validate_test.go

```diff
@@ -71,13 +71,13 @@ func TestValidate(t *testing.T) {
 		},
 		"openshift config invalid api fqdn": {
 			f: func(oc *api.OpenShiftManagedCluster) {
-				oc.Properties.FQDN = ""
+				oc.Properties.FQDN = "example.com"
 			},
-			expectedErrs: []error{errors.New(`invalid properties.fqdn ""`)},
+			expectedErrs: []error{errors.New(`invalid properties.fqdn "example.com"`)},
 		},
 		"test external only false - invalid fqdn fails": {
-			f:            func(oc *api.OpenShiftManagedCluster) { oc.Properties.FQDN = "()" },
-			expectedErrs: []error{errors.New(`invalid properties.fqdn "()"`)},
+			f:            func(oc *api.OpenShiftManagedCluster) { oc.Properties.FQDN = "example.com" },
+			expectedErrs: []error{errors.New(`invalid properties.fqdn "example.com"`)},
```

Comment:
validation tests should check if PublicHostname is set - fail. FQDN logic should be the same